packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -330,6 +330,10 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           )
         }
 
+        if (isExternalUrl(url)) {
```

Comment:
```suggestion
        if (isExternalUrl(resolved.id)) {
```

Should it be this? so it matches the previous behaviour.

Reply:
Done